AppImage-package/bundle.sh

```diff
@@ -20,12 +20,17 @@ cd $HOME/$APP/
 wget -q https://github.com/probonopd/AppImages/raw/master/functions.sh -O ./functions.sh
 . ./functions.sh
 
+URLS=$(apt-get install -qq --reinstall --print-uris  python-tk | cut -d "'" -f 2)
+wget $URLS
```

Comment:
Why are you downloading the .deb file then extracting it below, why not just copy the already installed files?